Solutions/PaloAltoPrismaCloud/Data Connectors/AzureFunctionPrismaCloud/main.py

```diff
@@ -21,12 +21,11 @@
 FILE_SHARE_CONN_STRING = os.environ['AzureWebJobsStorage']
 ALERT_LOG_TYPE = 'PaloAltoPrismaCloudAlert'
 AUDIT_LOG_TYPE = 'PaloAltoPrismaCloudAudit'
-
+AUDITLOGACCESS = os.environ['AuditLogAccess']
```

Comment:
@v-rucdu , Please provide default value, it is required for existing customer as they will not redeploy (in which case the default value  of Azure deploy does not work).